### ISSUE TITLE
skip, dryRun and prefix to command

### ISSUE THREAD

**Issue by Globegitter** [balderdashy/enpeem#3]
This is more a personal todo:
For Sane Stack I need a simple option to not actually run the npm-install and immediately return, an options to have a 'dry-run', which just saves the packages to the `package.json` without actually installing them, as well as prefixing something to the `npm install` command so it can be run in docker containers. That would be `docker-compose run server` for example.

Have the functionality for this locally in my code right now - will move that to enpeem with time :)

**Comment by sailsbot** [comment]
Thanks for posting, @Globegitter.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:
- review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
- create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
